scripts/integration-test

```diff
@@ -40,14 +68,25 @@ WEBHOOK_REPO=$(ctr image import ${IMAGE_FILE} | cut -d ' ' -f 2 | cut -d ':' -f
 source ./dist/tags
 
 # Install the webhook chart we just built.
-helm upgrade rancher-webhook ./dist/artifacts/rancher-webhook-${HELM_VERSION}.tgz -n cattle-system --set image.repository=${WEBHOOK_REPO} --set image.tag=${TAG} --reuse-values
-
-while ! kubectl rollout status -w -n cattle-system deploy/rancher-webhook; do
+# This command can fail since it is so close to the cluster start so we will give it 3 retires.
+RETRIES=0
+while ! helm upgrade rancher-webhook ./dist/artifacts/rancher-webhook-${HELM_VERSION}.tgz -n cattle-system \
+    --wait --set image.repository=${WEBHOOK_REPO} --set image.tag=${TAG} --reuse-values; do
+    if [ "$RETRIES" -ge 3 ]; then
+        exit 1
+    fi
+    RETRIES=$((RETRIES + 1))
     sleep 2
 done
 
 ./bin/rancher-webhook-integration.test -test.v -test.run IntegrationTest
 
+# Install the webhook chart with new ports.
+helm upgrade rancher-webhook ./dist/artifacts/rancher-webhook-${HELM_VERSION}.tgz -n cattle-system \
+    --wait --reuse-values --set port=443 --set capi.port=2319
```

Comment:
With the port being so low, should this also have `--set capNetBindService=true`?